README.md

```diff
@@ -111,7 +111,7 @@ The encode and protocol test suites does not require a
 running MySQL server on localhost.
 
 For the suites `mysql_tests`, `ssl_tests` and `transaction_tests` you need to
-start MySQL on localhost and give privileges to the user `otptest` and (for
+start MySQL on localhost and give privileges to the user `otptest` additionally you need to set my.cnf with `local_infile=1`. And (for
 `ssl_tests`) to the user `otptestssl`:
```

Comment:
This sentence is weird now. Better add your Additionally as a separate sentence after this one.